router/broker.go

```diff
@@ -28,6 +29,7 @@ var brokerRole = wamp.Dict{
 		featurePubExclusion:         true,
 		featurePubIdent:             true,
 		featureSubMetaAPI:           true,
+		featureTestamentMetaAPI:     true,
```

Comment:
According to the protocol specification, the feature is supposed to be announced by the _Dealer_ role:
https://wamp-proto.org/static/rfc/draft-oberstet-hybi-crossbar-wamp.html#feature-announcement-17

So, just move this in `dealer.go`

Reply:
I actually think this is a bug within the specification. Maybe @oberstet can help us here. 
Testament Meta API is listed within Advanced PubSub features, so I inferred to announce it within the broker. I will check against crossbario/crossbar reference implementation.

Reply:
Crossbar is compliant with the specification, and has `testament_meta_api` as a _Dealer_ feature:
https://github.com/crossbario/crossbar/blob/master/crossbar/router/dealer.py#L157

Reply:
To me it makes sense that is this a _Dealer_ feature since support or lack of does not affect what a client does with regard to the _Broker_, but it does affect what a client can do with regard to the _Dealer_, since this feature that is enabled or disabled is a meta-procedure.

Reply:
Changed it now, makes sense.